### ISSUE TITLE
Docs: Code snippet language picker missing required accessibility information

### ISSUE THREAD

**Issue by rschristian** [reduxjs/redux-toolkit#4790]
Just want to quickly drop mention as I happened to run into it tonight: the language picker (JS/TS) associated with most code snippets is missing the required accessibility props to tie a particular tab with a tabpanel. At a minimum, each tab should have `aria-controls` set w/ the ID of the corresponding tabpanel and you'll probably also want to add `aria-labelledby` to the tabpanel pointing back at the corresponding tab. Without this information, it's quite difficult for non-sighted users to discern the relationships between particular tabs and tabpanels.

[MDN has a very comprehensive example](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role#example) going over this, should be a quick & easy fix. Took quick peek myself but couldn't find where you were creating these components -- maybe it's a Docusaurus thing? Not sure.

**Comment by markerikson** [comment]
Yeah, this is just the `<TabItem>` that's built into Docusaurus.  Probably best to file an issue over there.
